### PR TITLE
Add EU Subnet docs

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/backend/eu-subnets.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+```
+
+## Important notes about deploying to European subnets
+
+When deploying a project to the mainnet using dfx:
+
+- If your canisters have already been created, dfx will continue to deploy the canisters on the same subnet that they are already located on.
+
+- If your canisters do not exist yet, when they are created dfx will create them on the same subnet as your wallet canister. It is recommended to consistently use a wallet on a European subnet.
```
Comment (sasa-tomic) [inline docs/developer-docs/backend/eu-subnets.md:101]:
```suggestion
- If your canisters do not exist yet, when they are created dfx will create them on the same subnet as your wallet canister. It is recommended to consistently use the wallet on the European subnet. 
```

File: docs/developer-docs/backend/subnet-types.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## System subnets
+
+System subnets are reserved for running canisters that provide a system functionality to ICP, such as the NNS canisters. System subnets have special configurations, such as not charging cycles to any of the canisters running on the subnet since system canisters need to be available at all times. System subnets also have a more generous per-call instruction limit and a more generous Wasm module size limit.
+
+## [European subnets](/docs/current/concepts/eu-subnets)
+
+European subnets are comprised of only node machines located in the European geographic region. This type of subnet allows for developers and enterprises to build applications that require a GDPR-aligned infrastructure and leverage blockchain decentralization with regional data sovereignty needs.
+
+
```
Comment (sasa-tomic) [inline docs/developer-docs/backend/subnet-types.md:23]: Would be worth mentioning the Fiduciary subnet as well? I don't know much about it though.

File: docs/developer-docs/backend/subnet-types.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+## Application subnets
+
+Application subnets are the most common type of subnet on ICP. Almost all canisters run on an application subnet. Application subnets can be individually configured to have different features enabled or disabled, for example the Bitcoin integration subnet. 
+
+## System subnets
+
+System subnets are reserved for running canisters that provide a system functionality to ICP, such as the NNS canisters. System subnets have special configurations, such as not charging cycles to any of the canisters running on the subnet since system canisters need to be available at all times. System subnets also have a more generous per-call instruction limit and a more generous Wasm module size limit.
+
+## [European subnets](/docs/current/concepts/eu-subnets)
```
Comment (sasa-tomic) [inline docs/developer-docs/backend/subnet-types.md:19]: Note that there is only one European subnet. There are no talks yet on adding a second one.

File: docs/developer-docs/backend/subnet-types.md (hunk 2 above)
Comment (jessiemongeon1) [reply docs/developer-docs/backend/subnet-types.md:31]: Yes, I have added a section that mentions Fiduciary subnets, using information from this forum post: https://forum.dfinity.org/t/introducing-the-first-fiduciary-subnet/17594/16

**Review by harpa12 — APPROVED** [review]

**Pull request merged**